Interview/ConductingATechnicalInterview.md

```diff
@@ -0,0 +1,26 @@
+Outline:
+
+How to Conduct a Technical Interview
+====================================
+
+### Remember what You Want to Achieve
+Best possible outcome from an interview is a new hire who makes great contributions to your organisation, your team and thrives as a member in your team. In this section I will try to say something about the mindset required to interview a candidate successfully.
+
+### Be prepared
+To not prepare for an interview is simply rude. How to best prepare does, however, depend on the kind of interview.
+
+### Make the Candidate Feel Welcome
+Discuss what we can do to make a probably nervous candidate feel welcome and respected.
+
+### Look for what the Candidate can bring
+Diverse teams are well known to be more productive so when interviewing we are interested to learn about experiences and knowledge that we currently don't have in the team. At the same time we need new hires to be able to work with the technology we are using. Realistically, candidates who bring in a wealth of knowledge about technology we are not using will not also be experts in the technology. How can we think about structuring an interview with these somewhat conflicting goals.
```

Comment:
```suggestion
Diverse teams are well known to be more productive, so when interviewing, we want to hear about their experiences and knowledge that we don't have in the team. 

But at the same time, we need new hires to be able to work with existing technology we are using. Realistically, candidates who bring in a wealth of knowledge about new technology only will NOT be considered as experts in our technology.

(TODO: Conclusion comes here. How do we actually want to measure these 2 different skills?
 Are both important? Or new/existing skills is more important than the other? 
 This needs to be documented, otherwise "But at the same time ..." paragraph will not be meaningful)
```

TODO needs to be answered, or just remove TODO and 2nd paragraph to make the whole section as simple as 1st paragraph only.